CHANGELOG.md

```diff
@@ -1,3 +1,38 @@
+CHANGES IN VERSION 1.10.0 (from 1.9.1)
+===================================
+
+Appium 1.10.0 is a minor release. 
+
+#### General
+* Bring Espresso Driver out of beta and into general availability
+* Support Xcode 10, discontinue support for Xcode 8
+* Improve string extractions for Android and iOS
+* Fix synchronization of applications caching [#274](https://github.com/appium/appium-base-driver/pull/274)
+
+#### iOS
+* Add [mobile:](http://appium.io/docs/en/commands/mobile-command/#ios-xcuitest-only) methods:
+  * Add biometrics (touchId, faceId) methods that can enroll/unenroll biometric features and send matching/non-matching biometric inputs (iOS Simulator only) [#816](https://github.com/appium/appium-xcuitest-driver/pull/816)
+  * Add method to clear keychains for an iOS Simulator (#816)[https://github.com/appium/appium-xcuitest-driver/pull/816]
+* Add [permissions capability](https://github.com/appium/appium-xcuitest-driver#desired-capabilities) that sets service permissions (calendar, siri, etc...) prior to creating a session [#818](https://github.com/appium/appium-xcuitest-driver/pull/818)
+* Add possibility to upload files into different container types on Simulator [#770](https://github.com/appium/appium-xcuitest-driver/pull/770)
+* Add [reduceMotion](https://github.com/appium/appium-xcuitest-driver/blob/master/README.md#L161) capability [#760](https://github.com/appium/appium-xcuitest-driver/pull/760)
+* Improve video recording
+* Improved iOS simulator booting procedure
+
+#### Android
+* Add [mobile:](http://appium.io/docs/en/commands/mobile-command) command for performing editor actions [#428](https://github.com/appium/appium-android-driver/pull/428)
+* Add `localeScript` capability to set script in `locale` (https://developer.android.com/reference/java/util/Locale) [#460](https://github.com/appium/appium-android-driver/pull/460)
+* Improved video recording
+* Fix [bug](https://github.com/appium/appium/issues/11619) calling `getRect` in MJSONWP sessions [#240](https://github.com/appium/appium-uiautomator2-driver/pull/240)
+* Fix pin entry error on Samsung devices that wasn't correctly locating the "Enter" button [#458](https://github.com/appium/appium-android-driver/pull/458)
+
+#### Android (UiAutomator2)
```

Comment:
Common Android change: it is not possible to record videos up to 30 min in length (requires ffmpeg to be installed on the system)